pkg/autoscaler/bucket/bucket.go

```diff
@@ -29,3 +33,19 @@ func IsBucketHost(host string) bool {
 	// have host with the prefix. Maybe use regexp for improvement.
 	return strings.HasPrefix(host, prefix)
 }
+
+// AutoscaleBucketName returns the name of the Autoscaler bucket with given `ordinal`
+// and `total` bucket count.
+func AutoscaleBucketName(ordinal, total uint32) string {
+	return strings.ToLower(fmt.Sprintf("%s-%02d-of-%02d", prefix, ordinal, total))
+}
+
+// AutoscalerBucketSet returns a point of hash.BucketSet consisting of Autoscaler
```

Comment:
```suggestion
// AutoscalerBucketSet returns a hash.BucketSet consisting of Autoscaler
```

("point" should be "pointer", but I dont think we normally comment on the return value being a pointer or not in comments, so might as well just drop it)

Reply:
dropped.